lrtable/src/lib/statetable.rs

```diff
@@ -127,7 +127,11 @@ where
                 .unwrap(),
             false
         );
-        let mut gotos = HashMap::with_hasher(BuildHasherDefault::<FnvHasher>::default());
+        let max = usize::from(grm.rules_len()) * usize::from(sg.all_states_len());
+        let mut gotos: Vec<StIdx> = Vec::with_capacity(max);
+        for _ in 0..max {
```

Comment:
I think you can replace the loop with `gotos.resize(max, StIdx::max_value())`.

Reply:
I'll try.